src/main/java/org/wso2/lsp4intellij/client/languageserver/requestmanager/DefaultRequestManager.java

```diff
@@ -87,13 +87,14 @@
 public class DefaultRequestManager implements RequestManager {
 
     private Logger LOG = Logger.getInstance(DefaultRequestManager.class);
+
+    public LanguageServerWrapper wrapper;
+    public LanguageServer server;
+    public LanguageClient client;
+    public ServerCapabilities serverCapabilities;
```

Comment:
If there's no special reason; shall we make these fields `private` and add some **getters** for these. I believe these values are being passed through the **constructor**. Allowing them `public` will make it harder to debug and test the code.

Reply:
Added :)